Sources/ComposableArchitecture/ViewStore.swift

```diff
@@ -45,7 +56,7 @@ import SwiftUI
 /// }
 /// ```
 ///
-/// > Important: The ``ViewStore`` class is not thread-safe, and all interactions with it (and the
```

Comment:
Did you mean to remove the double ticks here? Wasn't it intentional for DocC links? You may want to make these consistent across all doc comments either way, as I see a mix.